projects/frontend/shared-components/README.md

```diff
@@ -0,0 +1,16 @@
+# VDK Shared Components library
+
+This project implements common UI functionality that is project agnostic.
+The hope with having them in a separate directory is that other projects might leverage them.
+Within this project there is also a project(documentation-ui) for bootstrapping the individual screens for development and testing purposes.
+
+## Running
+
+Shared UI Components library is build and delivered as an Angular Library that is consumable via
+the NPMJS repository.
+See the `README.md` files in individual directories under `projects` for specific instructions.
+
+
+## Building, debugging, contributing and releasing
+
+See [CONTRIBUTING.MD](/CONTRIBUTING.md)
```

Comment:
todo edit the root CONTRIBUTING.md with `frontend` too 
https://github.com/vmware/versatile-data-kit/blob/main/CONTRIBUTING.md#structure-of-the-project